clippy_lints/src/methods/mod.rs

```diff
@@ -5182,7 +5182,7 @@ impl ShouldImplTraitCase {
 }
 
 #[rustfmt::skip]
-const TRAIT_METHODS: [ShouldImplTraitCase; 30] = [
+static TRAIT_METHODS: [ShouldImplTraitCase; 30] = [
```

Comment:
This change breaks the Clippy sync when testing the parallel compiler: https://github.com/rust-lang/rust/pull/131892

IIUC: With this being a `static`, it must implement `Sync`, but since `Span` doesn't implement `Sync` this also doesn't implement sync.

Reply:
Okay, for now we can revert back to `const` and add an allow if it fires. Longer term we should look at why Span isn't Sync.

Reply:
Done in https://github.com/rust-lang/rust/pull/131892/commits/cf918d56012663bcaace87b9876943fd5b683dd7